lib/modules/hosts/pixiv.js

```diff
@@ -0,0 +1,17 @@
+/* @flow */
+
+import { Host } from '../../core/host';
+
+export default new Host('pixiv', {
+	name: 'pixiv',
+	domains: ['pixiv.net'],
+	logo: '//www.pixiv.net/favicon.ico',
+	detect: ({ href }) => (/^https?:\/\/(?:www\.)?pixiv\.net\/member_illust\.php.*[?&]illust_id=(\d+)/i).exec(href),
+	handleLink(href, [, id]) {
+		const embed = `http://embed.pixiv.net/embed_mk2.php?id=${id}`;
+
+		return {
+			type: 'IFRAME',
```

Comment:
I believe you're going to need to add `embed,` to this object if you want to use that above URL.